css/onoffice_defaultview.css

```diff
@@ -258,6 +258,11 @@
 #oo-similarframe .oo-listobject {
 	width: inherit;
 }
+
+.oo-video .player-title {
```

Comment:
Do we break existing layouts with this?

Reply:
Same problem as https://github.com/onOfficeGmbH/oo-wp-plugin/pull/106#issuecomment-1025476719

What do you think about the possible solution?